lib/arelastic/queries/geo_shape/polygon.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Arelastic
+  module Queries
+    module GeoShape
```

Comment:
Created this module because GeoShape can [support many different GeoJSON ways to query](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html#input-structure). Wanted to stick as closely to the ES syntax as possible for naming.